README.md

```diff
@@ -2,32 +2,24 @@
 This module contains plugins and some example migrations to import data from a Fedora 3 Islandora instance
 into an Islandora CLAW instance.
 
-This is a base setup, it requires adjustments to the default Repository Object and configuration changes
-for your setup.
+This is a base setup, it requires configuration changes for your setup.
 
-## Required changes
-The default Repository Object provided with Islandora CLAW requires one additional field to allow for
-these migrations (or you can comment out these field migrations).
+## Installation and configuration
+To use this migration:
 
-1. A large text field called `field_mods_text`, this will store the MODS datastream from the source object.
+1. from within your Drupal 8 base directory (e.g. `/var/www/html/drupal`), run `composer require islandora/migrate_7x_claw`
+1. edit the .yaml files as indicated in the next section (it is important you edit these files before installing the module)
```

Comment:
Now that we're using a feature for the migration yml you can re-import the feature at any time.  So if you accidentally install the module first, or are actively developing your migration, you can iterate using drush or the UI to bring in changes.

Reply:
Yes, that would be useful. I ended up diabling then reenabling the two modules to refresh the config. Is there a specific command that does that?

Reply:
`drush -y fim islandora_migrate_7x_claw_feature` will re-import from the command line.

I don't know if there's one for cycling modules off and on again.  We went the features route to avoid the hassle of doing that.

Reply:
I'll add that to the README now. Thanks!